### PR TITLE
Add logs to matterbuild and small fix

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

**Review by jwilander — APPROVED** [review]
Probably would have been better to use a structured logger but this is fine for now

**Comment by cpanato** [comment]
@jwilander agree but for now which we dont see anything this is ok

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
```
Comment (crspeller) [inline Gopkg.toml]: Can we remove unnecessary constraints?

Comment (cpanato) [reply Gopkg.toml]: done

**Pull request merged**